bitnami/elasticsearch/templates/master/statefulset.yaml

```diff
@@ -240,6 +240,7 @@ spec:
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.master.livenessProbe "enabled") "context" $) | nindent 12 }}
             tcpSocket:
               port: rest-api
+
```

Comment:
Unrequired, right?

Reply:
Yes, it is just a 'Trigger' to force elasticsearch tests to be executed